sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/TPCDSQueryBenchmark.scala

```diff
@@ -44,9 +45,6 @@ import org.apache.spark.util.Utils
  */
 object TPCDSQueryBenchmark extends SqlBasedBenchmark with Logging {
 
-  private lazy val warehousePath =
-    Utils.createTempDir(namePrefix = "spark-warehouse").getAbsolutePath
```

Comment:
Why did we remove this?

Reply:
Do you proposes to reuse the stats and metastore between each run?

Reply:
What I actually would like to propose is not saving the data into a temporary location without table partitioning (https://github.com/apache/spark/pull/31218/files#diff-f0ef9be2f138cb947253f07c4285a8cf6b054355cf53beb2ba70ce82a380356bL77). I think we could simply use the input data as it is.
This line was added in https://github.com/apache/spark/pull/31011. I don't see why we need this temp dir after this PR.